Makefile

```diff
@@ -107,14 +110,14 @@ build-xdebug-72: vendor
 .PHONY: analyze cs-fix cs-check phpstan validate update
 analyze: cs-check phpstan validate
 
-cs-fix: build/cache php-cs-fixer-v2.phar
-	./php-cs-fixer-v2.phar fix -v --cache-file=build/cache/.php_cs.cache
+cs-fix: build/cache $(PHP-CS-FIXER)
+	$(PHP-CS-FIXER) fix -v --cache-file=build/cache/.php_cs.cache
 
-cs-check: build/cache php-cs-fixer-v2.phar
-	./php-cs-fixer-v2.phar fix -v --cache-file=build/cache/.php_cs.cache --dry-run --stop-on-violation
+cs-check: build/cache $(PHP-CS-FIXER)
```

Comment:
You can't depend on a variable if you can't *make* it. There's a rule for `php-cs-fixer-v2.phar`, make knows how to download it. But it can't possibly know how to download `php-cs-fixer-beta-v5.phar`. Build will fail.